react/MuiCozyTheme/makePalette.js

```diff
@@ -73,27 +108,27 @@ export const makePalette = type => {
     action: {
       active: getCssValue('actionColorActive'),
       hover: getCssValue('actionColorHover'),
-      hoverOpacity: 0.04,
+      hoverOpacity: get(opacityByTheme, `${type}.action.hoverOpacity`),
```

Comment:
Does this mean that we support `undefined` values?

I prefer the form `opacityByTheme[type].action.hoverOpacity` (this is subjective) but this will throw if the config is false. I suppose we should prefer that instead of having a silent error?

Reply:
nit: why is the parameter named `type` instead of `theme`?

Reply:
👍 pour `opacityByTheme[type].action.hoverOpacity` ! Non on ne supporte pas le undefined.

Pour le type plutôt que theme, c'est pour être cohérent avec l'argument passé dans la méthode générale makePalette... je vais regarder si c'est pertinent de refacto partout

Reply:
Also: instead you can avoid repetition by creating a local object like `const opacityByThemeConstants = opacityByTheme[type]` and then use it on all following lines.